nginx-redirects.conf

```diff
@@ -334,10 +334,18 @@ server {
 
 server {
     include /etc/nginx/ssl.default.conf;
+    include /etc/nginx/api-proxy.conf;
     server_name chimpandsee.org www.chimpandsee.org;
     return 301 https://www.zooniverse.org/projects/sassydumbledore/chimp-and-see;
 }
 
+server {
```

Comment:
We'll need to revert 454782fb67a64b5a7a2899a6884689d191820ac2 – it's not needed and it's actually creating a redirect loop for talk.chimpandsee.org:

```
$ curl -v -H "Host: talk.chimpandsee.org" localhost:8080/
...
< HTTP/1.1 301 Moved Permanently
...
< Location: http://talk.chimpandsee.org/
```